### PR TITLE
Remove no-display and multiple-display code where I can find it

### LINKED ISSUE #193: Change how SCARPE_DISPLAY_SERVICES works
Right now, SCARPE_DISPLAY_SERVICES is a list of names of display service classes. There's a lot wrong with that.

It would be nice to include some type of shorter nickname(s) -- wv for webview, glibui for GlimmerLibUI, things like that. It would be nice to not load display services that aren't being used (though separating them into gems would fix that.) Right now that's hard, though, because you have to load the files to find the class names, which means you have to load all display services to know for sure *what* classes are there.

Switching to nicknames that match the require and directory name would be an improvement.

### PR DESCRIPTION
**Pull request by noahgibbs**
Fix #193 and #202

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,11 +97,11 @@ Scarpe allows you to modify the app's behaviour outside of the normal Shoes API
...
 
-The SCARPE_DISPLAY_SERVICES environment variable allows you to choose one or more display services, from the default Webview service, to no service at all, to potentially other experimental or incomplete services. This may be important if you're developing a new display method for Scarpe. Normally ScarpeDisplayServices will contain a semicolon-delimited list of class names for display services (which can be just the name of a single display service). For no display service at all, set it to a single dash.
+The SCARPE_DISPLAY_SERVICE environment variable allows you to choose one or more display services, from the default Webview service to potentially other experimental or incomplete services. This may be important if you're developing a new display method for Scarpe. The display service variable will contain a name like "wv_local" or "wv_remote" or "glibui" which corresponds to a require-able Ruby file under lib/scarpe, either in the Scarpe gem or another gem your app requires.
 
 Example usage:
 
-`SCARPE_DISPLAY_SERVICES=Scarpe::GlimmerLibUIDisplayService ./exe/scarpe examples/hello_world.rb`
+`SCARPE_DISPLAY_SERVICE=glibui ./exe/scarpe examples/hello_world.rb`
```
Comment (Schwad) [inline README.md:104]: I didn't test out glimmerlibui manually locally but this will be a lot nicer approach to have

**Review by Schwad — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (noahgibbs) [reply README.md:104]: Yeah, I feel like the old way with the whole Ruby constant was kind of an embarrassment, but I didn't initially know what to do instead. I think this way is much better.

**Comment by Schwad** [comment]
Feel free to merge away when you're happy with the conflicts! I'd do it myself but that feels a bit more activist than merging alone with someone else's code 😅

**Comment by noahgibbs** [comment]
Will do! Conflict is fixed, waiting for CI to finish the tests.

**Pull request merged**